arcade/arcade/templates/{{ toolkit_name }}/CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# {{ toolkit_name }}
+
+{{ toolkit_description }}
+
+## Creating your new toolkit repo
+
+1. Run `arcade new` and answer all the questions
+2. Navigate to your new toolkit directory:
+   ```bash
+   cd {{ toolkit_name }}
+   ```
+3. Initialize git repository:
+   ```bash
+   git init
+   ```
+4. Create a new repository in Github with the same name as your new toolkit
+   > Note: Don't create a README, LICENSE or .gitignore when creating the repository because `arcade new` has already created these files for you.
+5. Add remote and push your code:
+   ```bash
+   git remote add origin https://github.com/{{ toolkit_author_name }}/{{ toolkit_name }}.git
+   git branch -M main
+   git add .
+   git commit -m "Initial commit"
+   git push -u origin main
+   ```
```

Comment:
Does this part make sense if `CONTRIBUTING.md` is created by `arcade new`? In other words, by the time I read this, I will have already completed all these steps.